lib/galaxy/jobs/mapper.py

```diff
@@ -76,47 +76,46 @@ def __invoke_expand_function(self, expand_function, destination):
             "referrer": destination,
         }
 
-        actual_args = {}
-
-        # Send through any job_conf.xml defined args to function
-        for destination_param in destination.params.keys():
-            if destination_param in function_arg_names:
-                actual_args[destination_param] = destination.params[destination_param]
-
-        # Populate needed args
-        for possible_arg_name in possible_args:
-            if possible_arg_name in function_arg_names:
-                actual_args[possible_arg_name] = possible_args[possible_arg_name]
-
         # Don't hit the DB to load the job object if not needed
         require_db = False
-        for param in ["job", "user", "user_email", "resource_params", "workflow_invocation_uuid"]:
+        for param in [
+            "job",
+            "user",
+            "user_email",
+            "resource_params",
+            "workflow_invocation_uuid",
+            "workflow_resource_params",
+        ]:
```

Comment:
This is a bug fix for workflow resource params right? Looks fine to me. Thanks.

Reply:
Potentially. Thanks for merging! @jmchilton